content/en/blog/2024/kubecon-china-2024.md

```diff
@@ -0,0 +1,56 @@
+---
+title:
+  OpenTelemetry Talks at Kubecon + CloudNativeCon + Open Source Summit + AI_dev
+  China 2024
+linkTitle: Kubecon China 2024
+date: 2024-06-25
+# prettier-ignore
+cSpell:ignore: Alhamdani Censhare EBPF Hrabusa Husni Huxing Jiahang Krom Sianturi Wanqi Zhang Zihao Ziyi
+author: '[Tiffany Hrabusa](https://github.com/tiffany76)'
+draft: true # TODO: remove this line once your post is ready to be published
+sig: Communications SIG
+---
+
+Join the OpenTelemetry project maintainers and members of the governance and
+technical committees at the [KubeCon + CloudNativeCon + Open Source Summit
++ AI_dev] conference in Hong Kong from August 21 to 23, 2024.
+
+Read on to learn about all the OpenTelemetry events at KubeCon China 2024.
+
+This post will be updated as we receive notice of other activities. Check back
+before the conference begins!
+
+## AI_dev talks
+
+- **[Adaptive Tracing Propagation with OpenTelemetry: Navigating Protocol Diversity in the Cloud | Project Lightning Talk](https://sched.co/1f4zX)**<br>
+  Wednesday, August 21 • 12:03 - 12:08
+- **[Telemetry API and OpenTelemetry: the answer of istio monitoring? | Project Lightning Talk](https://sched.co/1f4zz)**<br>
+  Wednesday, August 21 • 12:24 - 12:29
+
+## Maintainer sessions
+
+- **[OpenTelemetry Community Update | OpenTelemetry社区更新](https://sched.co/1eYcJ)**<br>
```

Comment:
@steverao, @ralf0131, @trask: thanks for organizing and running this! We (SIG Comms) would like you to announce our localization project for the website during this talk, let me know where/how we can quickly discuss this (slack or in a separate github issue)

Reply:
@austinlparker and I reviewed the slides earlier this week and they are planning to do this, @steverao @ralf0131 can you reach out to @svrnm on Slack to share the relevant slide and get any feedback from him? thanks!

Reply:
Awesome!